Source/UI/Controllers/AddressViewController.swift

```diff
@@ -12,8 +12,7 @@ public class AddressViewController: UIViewController,
     /// Address View
     public let addressView: AddressView! = AddressView(frame: .zero)
     let countrySelectionViewController = CountrySelectionViewController()
-    let doneButton = UIBarButtonItem(barButtonSystemItem: UIBarButtonItem.SystemItem.done,
-                                     target: self, action: nil)
+    let doneButton = UIBarButtonItem(barButtonSystemItem: UIBarButtonItem.SystemItem.done, target: nil, action: nil)
```

Comment:
When New UI is fully approved, I will ask Patrick to remove it.

Reply:
Are you saying this is currently in use? Because I tried commenting it out and it seems to compile for me. Its just that whenever we fix something (the compiler warning), we could spend an extra minute to ensure we don't fix it twice and whenever possible reduce lost code.

Reply:
TODO: is added for refactoring